nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/pom.xml

```diff
@@ -346,6 +346,12 @@
             <artifactId>jersey-spring5</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-test</artifactId>
+            <version>2.7.4</version>
```

Comment:
Currently only NiFi Registry contains Spring Boot dependencies. If this Spring Boot test dependency remains, recommend promoting the Spring Boot version from the NiFi Registry pom.xml to the root pom.xml for consistent versioning across the project.

Reply:
> Currently only NiFi Registry contains Spring Boot dependencies. If this Spring Boot test dependency remains, recommend promoting the Spring Boot version from the NiFi Registry pom.xml to the root pom.xml for consistent versioning across the project.

Sounds good.  I could definitely do that, if we decide to keep the test as is.

Reply:
> Is there a specific reason 2.7.4 was chosen? It appears that 2.7.4 depends on `spring-core` and `spring-context` 5.3.23 [1]. However, NiFi is already on `spring.version` 5.3.24. I think we bump to 2.7.6 which depends on 5.3.24.
> 
> https://repo1.maven.org/maven2/org/springframework/boot/spring-boot/2.7.4/spring-boot-2.7.4.pom

This could definitely be bumped, if we decide to go with things as is.